docs/developing/hardware-back-button.md

```diff
@@ -266,14 +266,15 @@ document.addEventListener('ionBackButton', (ev: BackButtonEvent) => {
 <TabItem value="angular">
 
 ```tsx
+import { Optional } from '@angular/core';
 import { IonRouterOutlet, Platform } from '@ionic/angular';
 import { App } from '@capacitor/app';
 
 ...
 
 constructor(
   private platform: Platform,
-  private routerOutlet: IonRouterOutlet
+  @Optional() private readonly routerOutlet?: IonRouterOutlet
```

Comment:
```suggestion
  @Optional() private routerOutlet?: IonRouterOutlet
```

Reply:
ok, no problem about not using `readonly`